server/models/images.js

```diff
@@ -12,3 +12,22 @@ module.exports.fetchAllByHostelId = (id, callback) => {
     callback(null, results);
   });
 };
+
+module.exports.createNewEntry = (newImage, callback) => {
+  // const queryStr = 'SELECT * FROM `images` WHERE hostel_id = ?';
+  // const queryArgs = [id];
+  const fileName = newImage.filename;
+  const url = newImage.url;
+  const description = newImage.description;
+  const hostelId = newImage.hostelID;
+  const queryStr = 'INSERT INTO `images` (file_name, url, description, hostel_id) VALUES (?, ?, ?, ?)';
+  const queryArgs = [fileName, url, description, hostelId];
+
+  db.query(queryStr, queryArgs, (error, results) => {
+    if (error || results.length === 0) {
```

Comment:
This error statement could be included in the express app and then you could just pass the top level callback straight down. 

`db.query(queryStr, queryArgs, callback)`